Gemfile

```diff
@@ -44,7 +33,7 @@ if RUBY_ENGINE == "ruby" and RUBY_VERSION > '1.9.2'
   gem 'puma'
   gem 'net-http-server'
   gem 'yajl-ruby'
-  gem 'nokogiri'
+  gem 'nokogiri', '~> 1.5'
```

Comment:
When I [tried a more elaborate versioning scheme](https://github.com/sinatra/sinatra/commit/a2eecf858a2ad4a844a57baf2b2d187c526ebb00) for versions 2.0 and below [didn't work](https://travis-ci.org/sinatra/sinatra/builds/192352440) and installed the wrong version of nokogiri.

Ignore the errors in that build about i18n, this was fixed by just pinning activesupport and i18n to a much older version that was still supported by everything. Although, admittedly this is really just a consequence of rabl having an open-ended dependency on AS.